new-log-viewer/src/services/LogFileManager.ts

```diff
@@ -175,6 +176,11 @@ class LogFileManager {
             } due to a limitation in Chromium-based browsers.`);
         }
 
+        const result = decoder.buildIdx(0, FULL_RANGE_END_IDX)
+        if (result != null && result.numInvalidEvents > 0) {
+            console.error(`${result.numInvalidEvents} were invalid.`);
```

Comment:
Would it be better to print the result out (as an object) directly so that we can better understand how many results are valid & how many are invalid?